onix/couple/couple_openmc.py

```diff
@@ -706,7 +706,10 @@ def _add_zero_dens_nuclides(self, cell):
         # Material is rename 'cell name' + 'mat'
         # New material id is 'mat id' + 'cell id'
```

Comment:
The comments should be updated accordingly to explain what happens.

Reply:
Done